commands/deploy_folder.js

```diff
@@ -102,8 +103,12 @@ exports.handler = async function (argv) {
             .originSign(originPrivateKey)
 
         try {
-            const { fee: fee } = await archethic.getTransactionFee(transaction, argv.endpoint)
-            console.log(chalk.yellow("Transaction fee : " +fee))
+            const { fee: fee, rates: rates } = await archethic.getTransactionFee(transaction, argv.endpoint)
+            
+
+            await yesno({
+                question:  chalk.yellow('The transaction would cost ' +fee+ ' UCO ($ ' +rates.usd+ ' € ' +rates.eur+ '). Do you want to confirm ?')
+            });
```

Comment:
I don't see you are saving the user input here. I mean, If he/she type `yes` or `no`. 
I don't see the logic for yes and no sections here...

If he/she press no means It should not allow transaction right? If yes, then it is not happening.

I feel like we are just prompting the message but not considering the user input any how.